WalletWasabi/Nostr/NostrExtensions.cs

```diff
@@ -0,0 +1,84 @@
+using System.Linq;
+using System.Net;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin.Secp256k1;
+using NNostr.Client;
+
+namespace WalletWasabi.Nostr;
+
+public static class NostrExtensions
+{
+	private const int Kind = 15750;
+	private const string TypeTagIdentifier = "type";
+	private const string TypeTagValue = "wabisabi";
+	private const string NetworkTagIdentifier = "network";
+	private const string EndpointTagIdentifier = "endpoint";
```

Comment:
This is Kukks' coding style. Remove it. See below how the code looks much cleaner without these constants.